packages/rbac/UpdateRole/index.ts

```diff
@@ -20,15 +20,15 @@ export type UpdateRoleParams = {
   uuid: string,
   /**
   * Update to a role
-  * @type { RoleWithAccess }
+  * @type { RolePut }
```

Comment:
@apinkert this makes me think in the future we may need to expand our executor to allow for an option to only build one of the many target open API specs we put in the `project.json` files. I'm a bit worried V2 is now coupled to changes like these in the main rbac client spec when we're really just wanting to test V2 changes only. It's not blocking - but I'll open an issue in our JS clients epic.